pkg/ocm/helpers.go

```diff
@@ -937,11 +939,19 @@ func ParseDiskSizeToGigibyte(size string) (int, error) {
 		return 0, fmt.Errorf("invalid disk size: %w", err)
 	}
 
+	if qty.Value() < 0 {
+		return 0, fmt.Errorf("invalid disk size: %s. positive size required", size)
```

Comment:
@thomasmckay I think we should add `'` around the placeholders in the string formats in this function to bring is into alignment with other functions. It also helps to catch whitespace problems. So ^ would become:

```
return 0, fmt.Errorf("invalid disk size: '%s'. positive size required", size)
```

Reply:
I'd be +1 on that but patterned this after the format just a few lines up. What do you think?

Reply:
I'd say let's make the other format line also use `'` around the placeholders, or does it look weird?

Reply:
done, thanks!